trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/CatalogConnectorManager.java

```diff
@@ -420,4 +421,26 @@ public void addMetalake(String metalake) {
   public Set<String> getUsedMetalakes() {
     return usedMetalakes;
   }
+
+  public Connector createConnector(
+      String connectorName, GravitinoConfig config, ConnectorContext context) {
```

Comment:
`context` has not been used, do you have other consideration that make it as a parameter?

Reply:
I will be used by next pr